Kentico.Kontent.Management.Tests/Unit/Data/ElementsData.cs

```diff
@@ -0,0 +1,211 @@
+using System;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Linq;
+using Kentico.Kontent.Management.Models.LanguageVariants;
+using Kentico.Kontent.Management.Models.LanguageVariants.Elements;
+using Kentico.Kontent.Management.Models.Shared;
+using Kentico.Kontent.Management.Modules.Extensions;
+
+namespace Kentico.Kontent.Management.Tests.Unit.Data
+{
+    /// <summary>
+    /// Represents dynamic and strongly typed objects of element data located in './ElementsData/ElementsData.json'.
+    /// </summary>
```

Comment:
I guess not :D I started adding them once the code was ready before the commit and it seems like I did not know when to stip :D

Reply:
But as for this one, I don't think it is necessary, but I believe it would help future devs to grasp the concept better. It would help me anyway... I'm on the edge here. Can we keep it?

Reply:
We can keep it then :)